lib/commands/certificate.js

```diff
@@ -0,0 +1,166 @@
+                    `Original value: ${JSON.stringify(altNames)}`);
+  }
+  const uuid = () => UUID.create().hex.toUpperCase();
+  const contentUuid = uuid();
+  return {
+    PayloadContent: [{
+      PayloadCertificateFileName: `${displayName}.cer`,
+      PayloadContent: cert,
+      PayloadDescription: 'Adds a CA root certificate',
+      PayloadDisplayName: displayName,
+      PayloadIdentifier: `com.apple.security.root.${contentUuid}`,
+      PayloadType: 'com.apple.security.root',
+      PayloadUUID: contentUuid,
+      PayloadVersion: 1
+    }],
+    PayloadDisplayName: displayName,
+    PayloadIdentifier: `localhost.${uuid()}`,
+    PayloadRemovalDisallowed: false,
+    PayloadType: 'Configuration',
+    PayloadUUID: uuid(),
+    PayloadVersion: 1
+  };
+}
+
+async function clickElement (driver, locator, timeout = 5000) {
+  let element = null;
+  try {
+    await waitForCondition(async () => {
+      try {
+        element = await driver.findNativeElementOrElements(
+          locator.type, locator.value, false);
+        return true;
+      } catch (ign) {
+        return false;
+      }
+    }, {
+      waitMs: timeout,
+      intervalMs: 500,
+    });
+  } catch (err) {
+    throw new Error(`Cannot find ${JSON.stringify(locator)} within ${timeout}ms timeout`);
+  }
+  await driver.nativeClick(element);
+}
+
+async function installCertificateInPreferences (driver) {
+  // Accept Safari alert
+  await clickElement(driver, {
+    type: 'accessibility id',
+    value: 'Allow'
+  });
+
+  await B.delay(2000);
+  // Go thought Preferences wizard
+  await clickElement(driver, {
+    type: 'accessibility id',
+    value: 'Install'
+  });
+  await B.delay(1500);
+  await clickElement(driver, {
+    type: 'accessibility id',
+    value: 'Install'
+  });
+
+  // Accept sheet alert
+  await clickElement(driver, {
+    type: '-ios class chain',
+    value: '**/XCUIElementTypeSheet/**/XCUIElementTypeButton[`label == \'Install\'`]'
+  });
+
+  // Finish adding certificate
+  await clickElement(driver, {
+    type: 'accessibility id',
+    value: 'Done'
+  });
+}
+
+/**
+ * @typedef {Object} CertificateInstallationOptions
+ *
+ * @property {!string} content - Base64-encoded content of the public certificate
+ */
+
+/**
+ * Installs a custom certificate onto the device.
+ * Since Apple provides no official way to do it via command line,
+ * this method tries to wrap the certificate into .mobileconfig format
+ * and then deploys the wrapped file to the internal HTTP server,
+ * so one can open it with mobile Safari.
+ * Then the algorithm goes through the profile installation procedure by
+ * clicking the necessary buttons using WebDriverAgent.
+ *
+ * @param {CertificateInstallationOptions} opts
+ */
+commands.mobileInstallCertificate = async function (opts = {}) {
+  const {content} = opts;
+  if (_.isEmpty(content)) {
+    throw new Error('Certificate content should not be empty');
+  }
+
+  const staticContentRoot = path.resolve(
```

Comment:
if we make this a real tested feature, we could probably expose the static root dir as part of the package so we don't need to go hunting for it